### PR TITLE
Update self-hosted docs for configuring the email identity availability

### PR DESCRIPTION
**Pull request by praneetloke**
Related to https://github.com/pulumi/pulumi-service/pull/7726.

### CODE REVIEW

**Review by chrsmith — APPROVED** [review]
The changes are fine, but it might be clearer to be explicit about "pairing" the console/API env vars.

It isn't super clear what is meant by "To hide the email signup option from the Console refer to []({{< relref "console#email-identity" >}})".

Instead, what do you think of a more imperative tone like: "This should only be set if the `PULUMI_DISABLE_EMAIL_SIGNUP` environment variable is set in the API, see [[configuring the api service]]."

Do you think that would be an improvement?

**Comment by praneetloke** [comment]
> It isn't super clear what is meant by "To hide the email signup option from the Console refer to []({{< relref "console#email-identity" >}})".

You beat me to it. I made a mistake there and didn't fill out the link text initially. I have fixed that now.

> Instead, what do you think of a more imperative tone like: "This should only be set if the `PULUMI_DISABLE_EMAIL_SIGNUP` environment variable is set in the API, see [[configuring the api service]]."

Sure, yeah. I'll update the text and let you know.

File: themes/default/content/docs/guides/self-hosted/console.md [hunk 1]
```diff
@@ -89,6 +89,13 @@ The following are the core environment variables that are required at a minimum.
 | BITBUCKET_OAUTH_ID| Atlassian Bitbucket OAuth consumer client ID. Used for Bitbucket OAuth signins. |
 | BITBUCKET_OAUTH_SECRET | |
 
+### Email and password {#email-identity}
+
+| Variable Name | Description |
+| ------------- | ----------- |
+| PULUMI_HIDE_EMAIL_LOGIN| When `true`, hides the email identity as a login option but does not disable the API handler. To disable the API handler for email logins set the API container environment variable `PULUMI_DISABLE_EMAIL_LOGIN`. Refer to the API container [environment variables]({{< relref "api#other-env-vars" >}}) for more information. |
```
Comment (cnunciato) [inline themes/default/content/docs/guides/self-hosted/console.md:96]:
```suggestion
| PULUMI_HIDE_EMAIL_LOGIN| When `true`, hides the email identity as a login option but does not disable the API handler. To disable the API handler for email logins set the API container environment variable `PULUMI_DISABLE_EMAIL_LOGIN`. Refer to the API container [environment variables]({{< relref ""/docs/guides/self-hosted/api#other-env-vars" >}}) for more information. |
```

File: themes/default/content/docs/guides/self-hosted/console.md [hunk 2]
```diff
@@ -89,6 +89,13 @@ The following are the core environment variables that are required at a minimum.
...
+| PULUMI_HIDE_EMAIL_SIGNUP | When `true` hides the email identity as a signup option but does not disable the API handler. To disable the API handler for email signups set the API container environment variable `PULUMI_DISABLE_EMAIL_SIGNUP`. Refer to the API container [environment variables]({{< relref "api#other-env-vars" >}}) for more information. |
```
Comment (cnunciato) [inline themes/default/content/docs/guides/self-hosted/console.md:97]:
```suggestion
| PULUMI_HIDE_EMAIL_SIGNUP | When `true` hides the email identity as a signup option but does not disable the API handler. To disable the API handler for email signups set the API container environment variable `PULUMI_DISABLE_EMAIL_SIGNUP`. Refer to the API container [environment variables]({{< relref ""/docs/guides/self-hosted/api#other-env-vars" >}}) for more information. |
```

**Comment by praneetloke** [comment]
@chrsmith I pushed an update. Can you please take another look? I didn't use the exact phrasing you had suggested. Instead, I made it explicit that simply hiding the button doesn't disable the API handler. That must be done separately.

**Review by cnunciato — APPROVED** [review]
Agree with Chris's feedback; otherwise just some `relref` fixups.

**Comment by praneetloke** [comment]
Thanks, @cnunciato! I have fixed up the relref's.

**Pull request merged**